### ISSUE TITLE
create a game setting for how 'lulzy' the game should be

### ISSUE THREAD

**Issue by stephenfournier** [stephenfournier/dotastrikers#153]
i'm honestly interested in adding lulzy stuff like natural disasters (tornadoes spawning randomly on the field), rune spawns, etc. there can be a game setting that allows the host to choose how 'lulzy' the game can be (like a simple slider lulzy -> competitive).

**Comment by CarpeSwag** [comment]
Yeah. We need a game options voting thing at the start (or just have them host enabled). I was going to focus on that soonish.
